control-net-qrcode/README.md

```diff
@@ -0,0 +1,60 @@
+# Control Net QR Code Truss
+
+This truss allows you to generate QR codes using Stable Diffusion and Control Net. By typing in a prompt using the same guidelines as stable diffusion, a new image gets created that combines the image from the prompt with the image of a qr code. For this truss [Stable Diffusion 1.5](https://huggingface.co/runwayml/stable-diffusion-v1-5) is used along with [Control QR Code Monster](https://huggingface.co/monster-labs/control_v1p_sd15_qrcode_monster) as the control net. 
+
+![controlnet_qr_code](controlnet_qr_code_results.gif)
+
+## Deployment
+
+First, clone this repository:
+
+```sh
+git clone https://github.com/basetenlabs/truss-examples/
+cd control-net-qrcode
+```
+
+Before deployment:
+
+1. Make sure you have a [Baseten account](https://app.baseten.co/signup) and [API key](https://app.baseten.co/settings/account/api_keys).
+2. Install the latest version of Truss: `pip install --upgrade truss`
+
+With `control-net-qrcode` as your working directory, you can deploy the model with:
+
+```sh
+truss push
+```
+
+Paste your Baseten API key if prompted.
+
+For more information, see [Truss documentation](https://truss.baseten.co).
+
+### Hardware notes
+
+This two models combined only take up about 5 GB of VRAM so a T4 is enough for this truss. 
```

Comment:
How fast is it on a T4? I'd probably recommend bumping the default to an A10G.

Reply:
It seems to be fast enough on the T4 so I don't think the A10G is needed, especially since most of the VRAM will be unused.